src/context.ts

```diff
@@ -1,13 +1,14 @@
 import path from "path";
 
-import { EventNames, EventPayloads, WebhookEvent } from "@octokit/webhooks";
+import { EventPayloads, WebhookEvent } from "@octokit/webhooks";
 import merge from "deepmerge";
 
 import type { Logger } from "pino";
 
 import { ProbotOctokit } from "./octokit/probot-octokit";
 import { aliasLog } from "./helpers/alias-log";
 import { DeprecatedLogger } from "./types";
+import { All } from "@octokit/webhooks/dist-types/generated/get-webhook-payload-type-from-event";
```

Comment:
We can use that as a temporary workaround, but we will have to export the events from `@octokit/webhooks` directly, otherwise we use internal APIs and it might break any moment. Before exporting though we should rename it to `Events` to make it more clear what it means outside of the context of `@octokit/webhooks`